goflags.go

```diff
@@ -341,6 +369,9 @@ func (stringSlice *StringSlice) createStringArrayDefaultValue() string {
 	return defaultBuilder.String()
 }
 
+// OtherOptionsGroupName is the name for all flags not in a group
+var OtherOptionsGroupName = "other options"
```

Comment:
This could be a constant instead.

Reply:
This is not made a constant to allow it to be changed, since we do want it to be customizable and not static. Just an optional change users of library can make